client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -21,19 +22,22 @@ class ConversationCaterpillarComponent extends React.Component {
 
 	render() {
 		const { comments, translate } = this.props;
-		const lastComment = last( comments );
-		const lastCommenterName = get( lastComment, 'author.name' );
-		const commentCount = comments.length;
+		const commentCount = size( comments );
```

Comment:
do we want to show actual comment count or just ones available to the :bug:?
I would think you actually want to use `post.discussion.comment_count`